docs/.style/fsdocs-custom.sass

```diff
@@ -41,6 +41,10 @@ $headings-font-family: 'Reem Kufi', sans-serif
 
 #fsdocs-content
   box-shadow: 0 4px 10px $drop-shadow
+  table.pre
+    background-color: $result-green
```

Comment:
Using a result color is a bit misleading here.
The output isn't linked to any of the results here so maybe go with something neutral:
$pearl background and $primary text maybe?